src/components/Widget/Widget.tsx

```diff
@@ -8,7 +8,7 @@ import DropdownMenu from 'components/DropdownMenu/DropdownMenu'
 import { useCloseWhenClickedOutside } from 'hooks/useCloseWhenClickedOutside'
 
 type WidgetProps = {
-  header: ReactNode
+  header?: ReactNode
```

Comment:
Suggestion: add a test in `src/components/Widget/Widget.test.tsx` that doesn't pass a header.